docs/_templates/search.html

```diff
@@ -13,17 +13,31 @@
 
     <div id="search-results">
     </div>
+
+    <nav id="pagination" class="pagination" aria-label="pagination" hidden="true" >
+        <button class="pagination-previous" id="pagination-previous">
+            <span><b>← Previous</b></span>
+        </button>
+        <ul class="pagination-list"></ul>
+        <button class="pagination-next" id="pagination-next">
+            <span><b>Next →</b></span>
+        </button>
```

Comment:
Those `<span>` tags don’t seem to do anything, and the `<b>` tags probably aren’t semantically appropriate inside a button. I think it’d be better to remove them:

```suggestion
        <button class="pagination-previous" id="pagination-previous">← Previous</button>
        <ul class="pagination-list"></ul>
        <button class="pagination-next" id="pagination-next">Next →</button>
```

Style-wise, in my opinion it’s good for the text styles of those next/previous buttons to be identical to the page buttons.